src/js/plugins/ajax.js

```diff
@@ -9,27 +9,15 @@ export default {
         data.SESSION_ID = store.getters.sessionID
       }
 
-      // For perl to accept the payload, we need to send it as form data
-      // so create a form data object to wrap the data in
-      const form = new FormData()
-      Object.keys(data).forEach(key => form.append(key, data[key]))
-
-      let config = {
-        headers: {'Content-Type': 'multipart/form-data' }
-      }
-      if (opt) {
-        config = {...config, ...opt}
-      }
-
-      return axios({
-        url,
-        method: 'post',
-        data: form,
-        header: {'Content-Type': 'multipart/form-data' }
-      })
+      // return axios({
```

Comment:
I think you can axe this chunk too rather than commenting out.

Reply:
OK, glad you checked this over since I really didn't know what I was doing.  I will delete it fully.